en-GB/px-platform.json

```diff
@@ -435,7 +435,7 @@
       },
       "practiceStudy": "If the study you’ve taken is a Practice Study, we’ll get in touch with you as soon as we review your results. Please remember that practice studies are not eligible for pay.",
       "moderatedSessionAboutToStart": {
-        "title": "You’re scheduled for a session at this time. \nTo continue, please log in to your account.",
+        "title": "You’re scheduled for a session at this time. \n To continue, please log in to your account.",
```

Comment:
Non-blocking/nit: Should this actually have a space at the beginning of a new line?

Reply:
Thanks, I missed that! Might be non-intentional, there are sometimes weird spacing issues with Phrase. Will double-check...

Reply:
Nevermind, just ran the "remove unusual line terminators" command on Visual Studio and the space disappeared - probably shouldn't have been there 🤷‍♀️